py_pdf_parser/visualise/main.py

```diff
@@ -74,35 +109,51 @@ def __init__(
             self.elements = document.elements
         self.show_info = show_info
 
-        page = document.get_page(current_page)
-        self.__fig, self.__ax = self.__initialise_plot(
-            width=page.width, height=page.height
+        self.root = root
+        if width is None:
+            width = self.root.winfo_screenwidth()
+        if height is None:
+            height = self.root.winfo_screenheight()
+        self.root.geometry(f"{width}x{height}")
+        title = f"py-pdf-parser"
```

Comment:
This doesn't need to be an f string.
```suggestion
        title = "py-pdf-parser"
```